src/harness/testcases/WINNF_FT_S_FDB_testcase.py

```diff
@@ -1078,18 +1086,15 @@ def test_WINNF_FT_S_FDB_7(self, config_filename):
 
     # Load the configuration file
     config = loadConfig(config_filename)
+    
+    # Very light checking of the config file.
+    self.assertEqual(len(config['expectedResponseCodes']), 2)
```

Comment:
This should be equal to the number of devices, not 2.

Reply:
FDB.7     Added:     self.assertEqual(len(config['registrationRequests']),   len(config['expectedResponseCodes']))